model-optimizer/mo/main.py

```diff
@@ -173,7 +182,7 @@ def raise_ie_not_found():
     if argv.legacy_ir_generation and len(argv.transform) != 0:
         raise Error("--legacy_ir_generation and --transform keys can not be used at the same time.")
 
-    use_legacy_fe = argv.framework not in new_front_ends
+    use_legacy_fe = argv.framework not in available_moc_front_ends
```

Comment:
Please add TODO about use_legacy_frontend option use in future becase this part will be changed once we introduce this option